src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -297,27 +277,46 @@ const RunPipelineDialog = (props: RunPipelineDialogProps) => {
                     </Field>
                   ))}
                 </div>
-                {form.submitError && (
-                  <div className="mt-3 text-sm text-red-600">
-                    {form.submitError}
-                  </div>
-                )}
-              </Dialog.Content>
-              <Dialog.Actions className="flex-1 items-center">
-                <div className="flex flex-1 items-center">
+                <Field
+                  name="notifiation"
+                  label={t("Notifications")}
+                  required
+                  className="mb-6"
+                >
                   <Checkbox
-                    checked={!form.formData.sendMailNotifications}
+                    checked={form.formData.sendMailNotifications}
```

Comment:
I believe you need to merge or rebase with the main branch, as this change will break the current behavior of sending notifications to recipients by default.

Reply:
Thanks for raising this. I just inverted the text and the value that is shown, so the behaviour logic should stay the same. It's confirmed by the unit tests : the boolean value has not changed

Reply:
Great, thx for the update !